format/wasm/mappers.go

```diff
@@ -0,0 +1,211 @@
+package wasm
+
+import (
+	"errors"
+
+	"github.com/wader/fq/pkg/scalar"
+)
+
+var sectionIDToSym = &sectionIDToSymMapper{}
+
+type sectionIDToSymMapper struct {
+}
+
+func (m *sectionIDToSymMapper) MapScalar(s scalar.S) (scalar.S, error) {
+	v, ok := s.Actual.(uint64)
+	if !ok {
+		return s, errors.New("unexpected data type for section ID")
+	}
+
+	switch v {
+	case sectionIDCustom:
+		s.Sym = "custom section"
+	case sectionIDType:
+		s.Sym = "type section"
+	case sectionIDImport:
+		s.Sym = "import section"
+	case sectionIDFunction:
+		s.Sym = "function section"
```

Comment:
I've usually tried to use string symbol values that look "codey", so no whitespace and maybe lowercase. But it might depend a bit on what the spec uses the names etc?

Reply:
Section IDs are defined in the specification like:
![image](https://user-images.githubusercontent.com/84613172/185551187-4df855c7-bb7e-4b33-94d8-ccff36e4214f.png)

But I agree that there is benefit in having the symbol values be ”codey". I'll change them to use underscores, not white spaces.

Reply:
👍  if you want a longer text not ment for queries etc then scalars values can have a description also